File: Largest Number/largest.js
var a = parseInt(prompt("Enter a value of a: "));
var b = parseInt(prompt("Enter a value of b: "));
var c = parseInt(prompt("Enter a value of c: "));

if(a>b & a>c){
    document.write(a +" is largest than " + b + " & " + c);
}
else if(b>a & b>c){
    document.write(b +" is largest than " + a + " & " + c);
}else{
    document.write(c +" is largest than " + a + " & " + b);
}

File: Largest Number/largest.py
num1 = int(input("Enter value of num1: "))
num2 = int(input("Enter value of num2: "))
num3 = int(input("Enter value of num3: "))

if(num1>num2 & num1>num3):
    print(f"{num1} is largest than {num2} & {num3}")
elif(num2>num1 & num2>num3):
    print(f"{num2} is largest than {num1} & {num3}")
else:
    print(f"{num3} is largest than {num1} & {num2}")